todoman/ui.py

```diff
@@ -91,7 +91,8 @@ def __init__(self, todo, databases, formatter):
         self._help_text = urwid.Text(
             '\n\nGlobal:\n'
             ' F1: Toggle help\n'
-            ' Ctrl-C: Cancel\n\n'
+            ' Ctrl-C: Cancel\n'
+            ' Ctrl-S: Save (only works if not a shell shortcut already)\n\n'
```

Comment:
At some point we probably should have configurable key bindings, I agree. However, consider that emacs uses `ctrl s` as well, despite this.